### DISCUSSION TITLE
Azure SQL Edge Retirement, replacement move to SQL 2022

### DISCUSSION THREAD

**Discussion by CarlSargunar** [umbraco/Umbraco-CMS#17261] [Features and ideas]
With the addition of the Dockerfile template, v14 now has docker support, but it uses Azure SQL Edge. Unfortunately microsoft is retiring that image, so support for Apple silicone is now through the latest SQL-2022 image

https://azure.microsoft.com/en-us/updates/v2/azure-sql-edge-retirement

The suggestion is to move the image over to using mcr.microsoft.com/mssql/server:2022-latest instead, which should be supported through Rosetta emulation on apple silicone. 

WIll probably need docs updated too, as well as the dockerfile

Happy to do these changes in a PR myself for hacktoberfest 👻🎃🧛‍♂️ but I thought I'd start a discussion first :-)

**Comment by CarlSargunar** [comment]
For ref : https://devblogs.microsoft.com/azure-sql/development-with-sql-in-containers-on-macos/

**Reply by CarlSargunar** [reply]
Draft PR Submitted : https://github.com/umbraco/Umbraco-CMS/pull/17262

**Comment by nikolajlauridsen** [comment]
Thanks for bringing this to our awareness 😄 

I agree with your suggestion to switch to the SQL server image. Especially since SQL Edge didn't work on macs, which takes away some of the value from the feature 😄 

I see you created an issue on the mssql-docker repo, interested to see what they come back with, however I do honestly think your current workaround is okay, if it means that we can get people on and running with Docker on Mac 😄
